dist/releases/4.10/config.toml

```diff
@@ -52,3 +52,13 @@ filter_files = [
   ### static binaries, no crypto
   "/checode-linux-libc/node_modules/@vscode/ripgrep/bin/rg"
 ]
+
+[payload.devspaces-udi-rhel8-container]
+filter_files = [
+  ### include golang compiler intentionally; exception needed until https://issues.redhat.com/browse/GT-130 is resolved
+  "/usr/lib/golang/pkg/tool/linux_amd64/compile",
+  "/usr/lib/golang/pkg/tool/linux_ppc64le/compile",
+  "/usr/lib/golang/pkg/tool/linux_s390x/compile",
+  ### helm is not FIPS compliant
+  "/usr/bin/helm"
```

Comment:
We should be able to rebuilt helm with the correct setting?

Reply:
sure but who is "we" and when will that happen?

Reply:
Can we remove it from the image?

Reply:
technically, yes. But afaik customers are using it.